sdcm/nemesis.py

```diff
@@ -3332,6 +3332,13 @@ def _run_commands_wait_and_cleanup(  # pylint: disable=too-many-arguments
                 self.log.debug(f"{name}: failed to execute cleanup command "
                                f"{cmd} on node {node} due to the following error: {str(exc)}")
 
+    def wait_restarted_node_fully_start(self, node, timeout=3600):
```

Comment:
better to rename to `wait_node_fully_start`

Reply:
maybe in the future, if moved into `cluster.py` it will make more sense, but since this function is here being used to wait to restarted nodes to fully start, the current name makes more sense.
fixed anyway